example/example_auth.dart

```diff
@@ -64,7 +65,21 @@ void _devices(SpotifyApi spotify) async =>
       print(devices.map((device) => device.name).join(', '));
     }).catchError(_prettyPrintError);
 
-void _prettyPrintError(Exception error) {
+void _createPrivatePlaylist(SpotifyApi spotify) async {
+  var user = await spotify.me.get();
+  await spotify.playlists
+      .createPlaylist(
+    user.id,
+    'Cool New Playlist 2',
+    description: 'Songs to test by',
+    public: false,
+  )
+      .then((playlist) {
+    print('Private playlist created!');
+  }).catchError(_prettyPrintError);
+}
+
+dynamic _prettyPrintError(Object error) {
```

Comment:
Do you need the `dynamic` here?😆

Reply:
Sorry, you're right, dynamic is not needed. But I did have to change `Exception` to `Object` for catchError to work.